webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsResponderADM.scala

```diff
@@ -177,47 +324,38 @@ final case class ProjectsResponderADM(actorDeps: ActorDeps, cacheServiceSettings
    * @param skipCache            if `true`, doesn't check the cache and tries to retrieve the project directly from the triplestore
    * @return information about the project as an optional [[ProjectADM]].
    */
-  def getSingleProjectADM(
+  override def getSingleProjectADM(
     identifier: ProjectIdentifierADM,
     skipCache: Boolean = false
-  ): Future[Option[ProjectADM]] =
-    tracedFuture("admin-get-project") {
+  ): Task[Option[ProjectADM]] = {
 
-      log.debug(
-        s"getSingleProjectADM - id: {}, skipCache: {}",
-        getId(identifier),
-        skipCache
-      )
-
-      for {
-        maybeProjectADM <-
-          if (skipCache) {
-            // getting directly from triplestore
-            getProjectFromTriplestore(identifier = identifier)
-          } else {
-            // getting from cache or triplestore
-            getProjectFromCacheOrTriplestore(identifier = identifier)
-          }
-
-        _ =
-          if (maybeProjectADM.nonEmpty) {
-            log.debug("getSingleProjectADM - successfully retrieved project: {}", getId(identifier))
-          } else {
-            log.debug("getSingleProjectADM - could not retrieve project: {}", getId(identifier))
-          }
+    logger.debug(
+      s"getSingleProjectADM - id: {}, skipCache: {}",
+      getId(identifier),
+      skipCache
+    )
 
-      } yield maybeProjectADM
-    }
+    for {
+      maybeProjectADM <-
+        if (skipCache) {
+          getProjectFromTriplestore(identifier = identifier)
+        } else {
+          // getting from cache or triplestore
+          getProjectFromCacheOrTriplestore(identifier = identifier)
+        }
+    } yield maybeProjectADM
+  }
 
   /**
    * Gets the project with the given project IRI, shortname, shortcode or UUID and returns the information
```

Comment:
Here is the same story with `UUID` for all the methods but easier would be to use replace tool. Feel free to do it or we can address it somewhere else.